frontend/src/app/modules/common/dynamic-forms/components/dynamic-form/dynamic-form.component.html

```diff
@@ -31,7 +31,7 @@
   class="op-form--fieldset"
   *ngIf="form && !handleSubmit"
   [form]="form"
-  [model]="innerModel"
+  [model]="_innerModel"
```

Comment:
Avoid using underscores for properties as per our style guide https://github.com/airbnb/javascript#naming--leading-underscore

Reply:
Ok, I didn't remember it, maybe we should adapt it because Typescript does have private variables. Here I wanted to emphasize that the proper model to use is the 'model' property. I'll change this.